policy_definitions/network_security/policy_definition_qby_deny_nsg_without_deny_vnetinboundtraffic_rule.json

```diff
@@ -0,0 +1,171 @@
+{
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "name": "QBY-Deny-NSG-Without-Deny-vNetInboundTraffic-Rule",
+  "properties": {
+    "displayName": "Deny Network Security Groups without default deny rule",
```

Comment:
Deny Network Security Groups without default deny "**intra vnet**" rule -> to be clear